packages/ipfs/test/http-api/inject/bootstrap.js

```diff
@@ -13,77 +13,118 @@ module.exports = (http) => {
     before(() => {
       api = http.api._httpApi._apiServers[0]
       return api.inject({
-        method: 'GET',
+        method: 'POST',
         url: '/api/v0/bootstrap/add/default'
       })
     })
 
-    it('/list', async () => {
-      const res = await api.inject({
-        method: 'GET',
-        url: '/api/v0/bootstrap/list'
+    describe('/list', () => {
+      it('only accepts POST', async () => {
+        const res = await api.inject({
+          method: 'GET',
+          url: '/api/v0/bootstrap/list'
+        })
+
+        expect(res.statusCode).to.equal(404)
```

Comment:
This is not interop test, but for what its worth, go-ipfs 0.5.0-rc1 returns [`405`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405):

```console
$ curl -I -X GET http://127.0.0.1:5001/api/v0/id
HTTP/1.1 405 Method Not Allowed
Allow: HEAD
Allow: OPTIONS
Allow: POST
Content-Type: text/plain; charset=utf-8
Vary: Origin
X-Content-Type-Options: nosniff
Date: Thu, 09 Apr 2020 23:48:22 GMT
Content-Length: 25
```

Reply:
That's the correct code but Hapi does not make this easy: 

https://github.com/hapijs/hapi/issues/1534
https://github.com/hapijs/hapi/issues/3238